### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by webrouse**
Solving: https://keboola.atlassian.net/browse/COM-365
Changes:
- Added [README.md](https://github.com/keboola/db-extractor-adapter/blob/67e99a0b98756d9b0b791453023a2e973bf79f4a/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
-# db-extractor-adapter
\ No newline at end of file
+# DB extractor adapter
+
+This library contains a common interface for connecting and data extraction from various sources:
```
Comment (pivnicek) [inline README.md:3]:
```suggestion
This library contains a common interface for connecting to, and extracting data from, various sources:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+    - By implementing this interface, it is possible to add support for CLI tools for export.
+    - Abstract class [`BaseExportAdapter`](https://github.com/keboola/db-extractor-adapter/blob/master/src/BaseExportAdapter.php) contains common code.
+    - Class [`PdoExportAdapter`](https://github.com/keboola/db-extractor-adapter/blob/master/src/PDO/PdoExportAdapter.php) implements export for PDO connection.
+    - Class [`OdbcExportAdapter`](https://github.com/keboola/db-extractor-adapter/blob/master/src/ODBC/OdbcExportAdapter.php) implements export for ODBC connection.
+    - Class [`FallbackExportAdapter`](https://github.com/keboola/db-extractor-adapter/blob/master/src/FallbackExportAdapter.php) allows you to use multiple adapters. If one fails, then fallback adapter is used.
+- **Interface [`QueryFactory`](https://github.com/keboola/db-extractor-adapter/blob/master/src/Query/QueryFactory.php)** servers to generate SQL query from `ExportConfig`. It is used if query is not set in the config.
```
Comment (pivnicek) [inline README.md:24]:
```suggestion
- **Interface [`QueryFactory`](https://github.com/keboola/db-extractor-adapter/blob/master/src/Query/QueryFactory.php)** used to generate SQL query from `ExportConfig`. It is used if query is not set in the config.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+    Class [`DefaultQueryFactory`](https://github.com/keboola/db-extractor-adapter/blob/master/src/Query/DefaultQueryFactory.php) is base implementation for MySQL/MariaDb compatible SQL dialects. 
+- **Class [`QueryResultCsvWriterTest`](https://github.com/keboola/db-extractor-adapter/blob/master/src/QueryResultCsvWriter.php)** servers to write rows from the `QueryResult` to the specified CSV file.
```
Comment (pivnicek) [inline README.md:26]: `QueryResultCsvWriterTest` Test?

**Comment by webrouse** [comment]
Thanks. I'm pretty weak in English.

**Comment by pivnicek** [comment]
not at all!  on the contrary, it's really good!

**Pull request merged**